src/modules/kvs/commit.c

```diff
@@ -480,7 +514,12 @@ commit_process_t commit_process (commit_t *c,
                 goto stall_load;
             }
 
-            if (!(c->rootcpy = json_copy (rootdir))) {
+            if (!treeobj_is_dir (rootdir)) {
```

Comment:
assertion appropriate here?  Looks like we (somehow) already prevent this from occurring.